src/main/kotlin/net/atos/client/zgw/ztc/model/ZaaktypeListParameters.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.client.zgw.ztc.model
+
+import jakarta.ws.rs.QueryParam
+import java.net.URI
+
+class ZaaktypeListParameters(
+    private val catalogus: URI
+) : AbstractZTCListParameters() {
+    /**
+     * URL-referentie naar de CATALOGUS waartoe dit ZAAKTYPE behoort.
+     */
+    @QueryParam("catalogus")
```

Comment:
I think you can use (similar as done on other fields) the annotation `@field:QueryParam("catalogus")` directly in the constructor, like so:
```
class ZaaktypeListParameters(
    @field:QueryParam("catalogus") val catalogus: URI
)
```

Reply:
No, afraid not. This is exactly what was generated (in some places) and then I found out this was causing the issue that the query params were not added to the requests.. I'm not completely sure why still but it really did not work.